osdk/src/cli.rs

```diff
@@ -114,20 +111,26 @@ pub struct TestArgs {
     pub osdk_args: OsdkArgs,
 }
 
-#[derive(Debug, Args, Default)]
+#[derive(Debug, Args, Default, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub struct CargoArgs {
     #[arg(
         long,
-        help = "Build artifacts in release mode",
-        default_value = "false"
+        help = "The Cargo build profile (built-in candidates are 'debug', 'release', 'dev' and 'bench')",
+        default_value = "dev"
     )]
-    pub release: bool,
+    pub profile: String,
```

Comment:
We allow users defining their own profiles. Just write it in the workspace Cargo.toml and OSDK will copy it for you to the base crate.

Reply:
The built in profile comment is fixed, and still the profile should be a string. It would be reported by cargo if user specified a invalid profile argument.